docs/gpu-sharing/mps/README.md

```diff
@@ -0,0 +1,37 @@
+# GPU Sharing with MPS implementation
+KAI Scheduler supports GPU sharing, allowing multiple pods to utilize the same GPU device efficiently by allocating a GPU device to multiple pods.
+
+The Multi-Process Service (MPS) is an alternative, binary-compatible implementation of the CUDA Application Programming Interface (API). The MPS runtime architecture is designed to transparently enable co-operative multi-process CUDA applications, typically MPI jobs, to utilize Hyper-Q capabilities on the latest NVIDIA (Kepler-based) Tesla and Quadro GPUs. (https://docs.nvidia.com/deploy/mps/index.html)
+
+There are several ways for enabling MPS on a Kubernetes cluster. In this document we will describe how to use MPS with KAI Scheduler.
+
+
+### Prerequisites
+1. KAI-Scheduler installed and running in your cluster, with gpu-sharing feature enabled.
+2. MPS Server up and running on all host with GPUs (`nvidia-cuda-mps-control`) configured with `CUDA_MPS_PIPE_DIRECTORY` to `/tmp/nvidia-mps`
+
+### MPS Enabled PODs
+To submit a pod that can share a GPU device, and is connecting to the MPS server run this command:
+```
+kubectl apply -f gpu-sharing-with-mps.yaml
+```
+
+In the gpu-sharing-with-mps.yaml file, the pod includes a `mps` volume with hostPath of `/tmp/nvidia-smp`, that is mounted to the container to same path.
+
```

Comment:
Suggestion:
In the gpu-sharing-with-mps.yaml file, the pod defines an mps volume using a hostPath set to `/tmp/nvidia-smp`, which is mounted to the same path within the container.

Reply:
Done